### PR TITLE
BXMSDOC-5870: Added Boxed List expression section and updated Decision navigator view screenshot

### PR DESCRIPTION
**Pull request by hmanwani-rh**
See JIRA: https://issues.redhat.com/browse/BXMSDOC-5870

Epic: https://issues.redhat.com/browse/BXMSDOC-5602

Rendered output
[Designing a decision service using DMN models RHPAM](http://file.pnq.redhat.com/~hmanwani/BXMSDOC-5870-RHPAM/#dmn-list-expressions-con_dmn-models)
[Designing a decision service using DMN models RHDM](http://file.pnq.redhat.com/~hmanwani/BXMSDOC-5870-RHDM/#dmn-list-expressions-con_dmn-models)

### CODE REVIEW

**Review by karreiro — APPROVED** [review]
Thank you, @hmanwani-rh.

File: doc-content/drools-docs/src/main/asciidoc/DMN/dmn-list-expressions-con.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+= Boxed list expressions
+
+A boxed list expression in DMN represents a FEEL list of items. You use boxed lists to define lists of relevant items for a particular node in a decision. You can also use literal FEEL expressions for list items in cells to create more complex lists.
+
+For example, the following boxed list expression identifies approved credit score agencies in a loan application decision service:
+
+.Boxed list expression for approved credit score agencies
+image::dmn/dmn-list-expression-example.png[]
+
+The following boxed list expression also identifies approved credit score agencies but uses FEEL logic to define the agency status (Inc., LLC, SA, GA) based on a DMN input node:
```
Comment (csherrar) [inline doc-content/drools-docs/src/main/asciidoc/DMN/dmn-list-expressions-con.adoc:11]: Does the FEEL logic for adding the suffix include the comma or should the comma be added? For example: `"Acme Agency," + suffix

**Review by csherrar — COMMENTED** [review]
Just a question, otherwise looks good to me

File: doc-content/drools-docs/src/main/asciidoc/DMN/dmn-list-expressions-con.adoc (hunk 1 above)
Comment (hmanwani-rh) [reply doc-content/drools-docs/src/main/asciidoc/DMN/dmn-list-expressions-con.adoc:11]: @csherrar, thanks for raising the question. 
As discussed with the SME, the syntax for FEEL logic is string + another string, and in the above example the value of suffix string is `, Inc.`.

**Review by jomarko — APPROVED** [review]

**Pull request merged**